### ISSUE TITLE
FovVideoVDP as a loss function?

### ISSUE THREAD

**Issue by tvkamara** [gfxdisp/FovVideoVDP#1]
Greetings and thanks for a great new metric. Seems to be just what is needed on the field. In the paper you mention that in the future you plan to "use the metric as a differentiable loss function". I would be interested to do just this. I'm curious if you have insights into what (if any) constraints there are currently in the loss code that would prevent this. I guess this comes down to the question if the current implementation (vdploss = FovVideoVDP.load( ...) is differentiable or not?

BR, Teemu

**Comment by achapiro** [comment]
Hi Teemu,
Thank you for reaching out!
The current Pytorch implementation of the metric is differentiable and can be used as a loss function, but in our early tests it didn't perform well for this purpose. There could be several reasons for this (e.g. issues with stability).
As you noted, in our paper we mention an extension to a differentiable loss function as future work.
Best,
Alex.
